docs/reference/components/sinks/redis.cue

```diff
@@ -0,0 +1,127 @@
+package metadata
+
+components: sinks: redis: {
+	title: "Redis"
+	classes: {
+		commonly_used: false
+		delivery:      "best_effort"
+		development:   "beta"
+		egress_method: "stream"
+		service_providers: []
+		stateful: false
+	}
+	features: {
+		buffer: enabled:      false
+		healthcheck: enabled: true
+		send: {
+			compression: enabled: false
```

Comment:
I think we could add `batch: enabled: false` here as well since this sink doesn't support batching.

Reply:
if add `batch: enabled: false` , an error occurs when executing the command `make check-docs`

Reply:
> I think we could add `batch: enabled: false` here as well since this sink doesn't support batching.

if add `batch: enabled: false` , an error occurs when executing the command `make check-docs`